### PR TITLE
docs: tweaks to tutorial based on user feedback

### PR DESCRIPTION
**Pull request by afgambin**
Addressing user feedback and this [issue](https://github.com/canonical/data-science-stack/issues/152).

### CODE REVIEW

File: docs/tutorial/getting-started.rst [hunk 1]
```diff
@@ -15,6 +15,10 @@ Prerequisites
 * Ubuntu 22.04 LTS.
 * An Internet connection.
 * `Snap`_ installed.
+* 50GB of disk space is recommended. This includes the `requirements <https://microk8s.io/docs/getting-started>`_ for MicroK8s.
+
+.. note::
+   All the code commands provided throughout this guide can be run from any host terminal.
```
Comment (afgambin) [inline docs/tutorial/getting-started.rst:21]: Is this correct?

File: docs/tutorial/getting-started.rst [hunk 2]
```diff
@@ -15,6 +15,10 @@ Prerequisites
 * Ubuntu 22.04 LTS.
 * An Internet connection.
 * `Snap`_ installed.
+* 50GB of disk space is recommended. This includes the `requirements <https://microk8s.io/docs/getting-started>`_ for MicroK8s.
```
Comment (afgambin) [inline docs/tutorial/getting-started.rst:18]: I added this info here, as part of the prerequisites.

File: docs/tutorial/getting-started.rst [hunk 3]
```diff
@@ -64,7 +63,13 @@ Next, you need to initialise DSS on top of MicroK8s and prepare MLflow:
 
    The initialisation might take a few minutes to complete.
    While the process is in progress, you will see the following message:
-   ``[INFO] Waiting for deployment my-tensorflow-notebook in namespace dss to be ready...``
+   ``[INFO] Waiting for deployment mlflow in namespace dss to be ready...``
```
Comment (afgambin) [inline docs/tutorial/getting-started.rst:66]: Got this feedback from a user, before it was wrong. First time you initialize, you see `mlflow`. Once you create a notebook, then `Waiting for deployment <your_notebook>` appears.

File: docs/tutorial/getting-started.rst (hunk 2 above)
Comment (kimwnasptd) [reply docs/tutorial/getting-started.rst:18]: Looks good!

File: docs/tutorial/getting-started.rst (hunk 1 above)
Comment (kimwnasptd) [reply docs/tutorial/getting-started.rst:21]: The note feels a bit weird to me when I read it initially. What do we want to convey?

File: docs/tutorial/getting-started.rst (hunk 3 above)
Comment (kimwnasptd) [reply docs/tutorial/getting-started.rst:66]: Ah good point, it should be mlflow initially

File: docs/tutorial/getting-started.rst (hunk 1 above)
Comment (afgambin) [reply docs/tutorial/getting-started.rst:21]: @kimwnasptd  The place where the user is expected to run the tutorial from. Is that info trivial/assumed? If so, we can remove it.

Thinking, for instance, a non-linux data scientist, coming from Windows, not used to run stuff from a CLI but using UIs.

Comment (kimwnasptd) [reply docs/tutorial/getting-started.rst:21]: I think in this case it's trivial and would suggest to drop :)

**Comment by afgambin** [comment]
@kimwnasptd it should be good to go now. If you can pls approve and merge.

**Review by kimwnasptd — APPROVED** [review]

**Comment by kimwnasptd** [comment]
Thanks for picking this up @afgambin!

**Pull request merged**